CharlieBackend.Api.UnitTes/CurrentUserTests.cs

```diff
@@ -0,0 +1,97 @@
+using CharlieBackend.Business.Services;
+
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class CurrentUserTests : TestBase
+    {
+        [Fact]
+        public void ThrowErrorIfNotLoggined()
+        {
+            var _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            _httpContextAccessorMock.Setup(req => req.HttpContext)
+                .Returns(new DefaultHttpContext());
+            var currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            Assert.Throws<UnauthorizedAccessException>(() => currentuserService.AccountId);
+        }
+
+        [Fact]
+        public void getAccountId()
+        {
+            var _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            var httpContext = new DefaultHttpContext();
+            var mockIdentity = new Mock<GenericIdentity>("Vasya");
+            mockIdentity.Setup(i => i.Claims)
+                .Returns(new Claim[]{
+                    new Claim("AccountId", "123") 
+                });
+            httpContext.User = new ClaimsPrincipal(mockIdentity.Object);
+            _httpContextAccessorMock.Setup(req => req.HttpContext).Returns(httpContext);
+            var currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            var Id = currentuserService.AccountId;
+
+            Assert.Equal(123,Id);
+        }
+        [Fact]
+        public void getEntityId()
+        {
+            var _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            var httpContext = new DefaultHttpContext();
+            var mockIdentity = new Mock<GenericIdentity>("Vasya");
+            mockIdentity.Setup(i => i.Claims)
+                .Returns(new Claim[] { 
+                    new Claim("Id", "123") 
+                });
+            httpContext.User = new ClaimsPrincipal(mockIdentity.Object);
+            _httpContextAccessorMock.Setup(req => req.HttpContext).Returns(httpContext);
+
+            var currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            var EntityId = currentuserService.EntityId;
+
+            Assert.Equal(123, EntityId);
+        }
+        [Fact]
+        public void getEmail()
+        {
+            var _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            var httpContext = new DefaultHttpContext();
+
+            var mockIdentity = new Mock<GenericIdentity>("Vasya");
+            mockIdentity.Setup(i => i.Claims)
+                .Returns(new Claim[] {
+                    new Claim("Email", "Hello@mail.com") 
+                });
+            httpContext.User = new ClaimsPrincipal(mockIdentity.Object);
+            _httpContextAccessorMock.Setup(req => req.HttpContext).Returns(httpContext);
+
+            var currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            var Email = currentuserService.Email;
+
+            Assert.Equal("Hello@mail.com", Email);
```

Comment:
Also I think we're supposed to use **FluentAssertions** for tests. You can view `MentorServiceTests` for reference.

e.g.:
https://github.com/ita-social-projects/WhatBackend/blob/899f397be4e51f81fc156f059c3ced0a0a60d669/CharlieBackend.Api.UnitTes/MentorServiceTests.cs#L65

Reply:
Solved in https://github.com/ita-social-projects/WhatBackend/pull/504/commits/e24ea4bfff6c941a77091f4f006367542332ab6b